### PR TITLE
ACS-8124: Rename package prediction_applier to hxi_extension

### PR DESCRIPTION
**Pull request by krdabrowski**
https://hyland.atlassian.net/browse/ACS-8124

### CODE REVIEW

**Review by mpichura — APPROVED** [review]
LGTM

File: prediction-applier-extension/src/test/java/org/alfresco/hxi_connector/hxi_extension/service/PredictionServiceImplTest.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ PROP_PREDICTION_DATE_TIME, new Date(1_234_567_890L),
                 PROP_MODEL_ID, "hx-model-id",
                 PROP_PREDICTION_VALUE, "blue",
                 PROP_PREVIOUS_VALUE, "red",
-                PROP_UPDATE_TYPE, AUTOCORRECT.name(),
+                PROP_UPDATE_TYPE, UpdateType.AUTOCORRECT.name(),
```
Comment (tpage-alfresco) [inline prediction-applier-extension/src/test/java/org/alfresco/hxi_connector/hxi_extension/service/PredictionServiceImplTest.java:135]: We lost a static import here.

**Review by tpage-alfresco — APPROVED** [review]
LGTM

File: prediction-applier-extension/src/test/java/org/alfresco/hxi_connector/hxi_extension/service/PredictionServiceImplTest.java (hunk 1 above)
Comment (krdabrowski) [reply prediction-applier-extension/src/test/java/org/alfresco/hxi_connector/hxi_extension/service/PredictionServiceImplTest.java]: Fixed

**Pull request merged**